build with postgresql as module is broken

```
  CXXLD    dnspcap2protobuf
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQL::~SPgSQL()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:323: undefined reference to `PQfinish'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQL::sPerrorException(std::string const&)':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:328: undefined reference to `PQerrorMessage'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQL::execute(std::string const&)':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:333: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:334: undefined reference to `PQresultStatus'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:335: undefined reference to `PQresultErrorMessage'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:336: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQL::SPgSQL(std::string const&, std::string const&, std::string const&, std::string const&, std::string const&, std::string const&)':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:301: undefined reference to `PQconnectdb'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:303: undefined reference to `PQstatus'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:309: undefined reference to `PQfinish'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::reset()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:184: undefined reference to `PQclear'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:186: undefined reference to `PQclear'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:179: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:180: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::releaseStatement()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:215: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:216: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::reset()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:184: undefined reference to `PQclear'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:186: undefined reference to `PQclear'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:179: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:180: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::releaseStatement()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:215: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:216: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::reset()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:184: undefined reference to `PQclear'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:186: undefined reference to `PQclear'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:179: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:180: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::releaseStatement()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:215: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:216: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::execute()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:89: undefined reference to `PQexecPrepared'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:90: undefined reference to `PQresultStatus'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:91: undefined reference to `PQresultErrorMessage'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::nextResult()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:104: undefined reference to `PQntuples'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:112: undefined reference to `PQftype'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:133: undefined reference to `PQntuples'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:134: undefined reference to `PQnfields'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:105: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::execute()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:85: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:86: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::prepareStatement()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:225: undefined reference to `PQprepare'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:226: undefined reference to `PQresultStatus'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:227: undefined reference to `PQresultErrorMessage'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:228: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::nextResult()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:120: undefined reference to `PQgetvalue'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:126: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:127: undefined reference to `PQntuples'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:128: undefined reference to `PQnfields'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::execute()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:93: undefined reference to `PQresultErrorMessage'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::prepareStatement()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:225: undefined reference to `PQprepare'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:226: undefined reference to `PQresultStatus'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:227: undefined reference to `PQresultErrorMessage'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:228: undefined reference to `PQclear'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:225: undefined reference to `PQprepare'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:226: undefined reference to `PQresultStatus'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:227: undefined reference to `PQresultErrorMessage'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:228: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::nextRow(std::vector<std::string, std::allocator<std::string> >&)':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:147: undefined reference to `PQnfields'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:148: undefined reference to `PQnfields'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:149: undefined reference to `PQgetisnull'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:151: undefined reference to `PQftype'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:155: undefined reference to `PQgetvalue'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:160: undefined reference to `PQclear'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::nextResult()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:104: undefined reference to `PQntuples'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:112: undefined reference to `PQftype'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:120: undefined reference to `PQgetvalue'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:126: undefined reference to `PQexec'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:127: undefined reference to `PQntuples'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:128: undefined reference to `PQnfields'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::nextRow(std::vector<std::string, std::allocator<std::string> >&)':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:152: undefined reference to `PQgetvalue'
../modules/gpgsqlbackend/.libs/spgsql.o: In function `SPgSQLStatement::nextResult()':
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:133: undefined reference to `PQntuples'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:134: undefined reference to `PQnfields'
/home/kees/src/pdns/modules/gpgsqlbackend/spgsql.cc:105: undefined reference to `PQclear'
collect2: error: ld returned 1 exit status
make[3]: *** [pdnsutil] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory `/home/kees/src/pdns/pdns'
make[2]: *** [all] Error 2
make[2]: Leaving directory `/home/kees/src/pdns/pdns'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/kees/src/pdns'
make: *** [all] Error 2
```

minimal reproduction master (el7)

```
 ./bootstrap; ./configure --with-modules='gpgsql' --with-dynmodules=''; make
```

--- a/modules/gpgsqlbackend/OBJECTLIBS
+++ b/modules/gpgsqlbackend/OBJECTLIBS
@@ -1,1 +1,1 @@
--lssl -lcrypto $(PGSQL_lib)
+-lssl -lcrypto $(PGSQL_LIBS)

